.studio/automate-gateway

```diff
@@ -548,29 +548,23 @@ document "get_api_token_with_policy" <<DOC
   ----------------------------------------------------
   # get_api_token_with_policy editor-access
 
-  Note: if NOT using IAM v2, this token will just be an admin token.
 DOC
 function get_api_token_with_policy() {
   check_if_deployinate_started || return 1
 
-  local policy=${1:?"usage: get_api_token_with_policy POLICYNAME"}
-  local token_description="api-token-$$-${policy}"
-  local token_file="/tmp/api_token_${policy}"
+  local policy_id=${1:?"usage: get_api_token_with_policy POLICYID"}
+  local token_id="api-token-id-${policy_id}"
+  local token_file="/tmp/api_token_${policy_id}"
 
   if [ -f "$token_file" ]; then
     cat "$token_file"
   else
     # create standard api token
-    chef-automate iam token create "$token_name" >"$token_file" || return 1
-    # retrieve the tokens' ID
-    local token_id
-    token_id="$(curl -fsS --insecure -H "api-token: $(get_admin_token)" \
-      "${GATEWAY_URL}/iam/v2/tokens" |\
-        jq -er --arg token "$token_name" '[.tokens[] | select(.name == $token) | .id] | first' )"
+    chef-automate iam token create "$token_id" --id "$token_id" >"$token_file" || return 1
```

Comment:
changes to this file fix the `iam automate-gateway integration` test in the verify_private pipeline